### ISSUE TITLE
boot version ... target creates additional tree in target

### ISSUE THREAD

**Issue by jgrzebyta** [degree9/boot-semver#42]
Running more than once `boot version` ... `target` crates some weird tree `target/target/..../target/version.properties`.

After a few runs I have:

```
target/
├── logback.xml
├── target
│   ├── target
│   │   ├── target
│   │   │   ├── target
│   │   │   │   └── version.properties
│   │   │   └── version.properties
│   │   └── version.properties
│   └── version.properties
└── version.properties
```

**Comment by burn2delete** [comment]
This is because the version task explicitly gets its location from the directory boot was launched in, I'll see about what can be done here.

**Comment by burn2delete** [comment]
Also which version are you running?

**Comment by jgrzebyta** [comment]
The latest one: 1.3.6

**Comment by burn2delete** [comment]
Could you provide the boot file which is having the issue?

**Comment by jgrzebyta** [comment]
Apologies for delay.
This occurs in the latest version of my project. Build file in [sesame-loader](https://raw.githubusercontent.com/jgrzebyta/sesame-loader/master/build.boot).

**Comment by jgrzebyta** [comment]
This error occurs in the library version `1.4.1`.

**Comment by burn2delete** [comment]
What command are you running? I cannot reproduce this issue.

**Comment by burn2delete** [comment]
could it be the double call to `jar` in your build task?

**Comment by jgrzebyta** [comment]
I run for example `boot version target` several times. My building task code is:
```
(deftask build-standalone
  "Build standalone version" []
  (comp
   (version)
   (pom)
   (aot)
   (uber)
   (jar :file (format "%s-standalone.jar" (name (get-env :project))))
   (target)))
```

When I remove `version` subtask then boot cried: `need project and version to create pom.xml` because version is not declared in separate variable, eg. `+version+` but is fully managed by the plugin.

task options are:
```
(defn current [& _] "16")

(task-options!
 version {:minor 'one :patch 'current :pre-release 'snapshot :include true}
 pom {:project (get-env :project) }
 aot {:namespace '#{rdf4j.loader rdf4j.sparql rdf4j.dump}})
```

**Comment by burn2delete** [comment]
Can you test this on another project, I was not able to build your project to debug this. My test project seems to work without issues.

**Comment by jgrzebyta** [comment]
@flyboarder  I built mock project (attached) with example code and the error occurs:

After run `boot build-standalone` several times there is following tree inside target directory (you can find it also inside attached file):

![tree](https://cloud.githubusercontent.com/assets/6738268/23262827/e67f6024-f9d3-11e6-8a50-5a7c79c877d7.png)


[mock-issue-42.tar.gz](https://github.com/degree9/boot-semver/files/796658/mock-issue-42.tar.gz)

**Comment by burn2delete** [comment]
@jgrzebyta best bug report ever! ⭐️  compressed mock repo and a screenshot!

Can you please give version `1.4.2` a shot
EDIT: nvm that also has the issue, i know where it's coming from now, thanks a bunch!

**Comment by burn2delete** [comment]
@jgrzebyta Finally squashed in `1.4.3`, please confirm.

**Comment by jgrzebyta** [comment]
Thanks a lot. :+1: I found it is fixed now in `1.4.3`.
